src/middleware/checkers.js

```diff
@@ -0,0 +1,26 @@
+class Checkers {
```

Comment:
Hello @kechyy, can we have this file renamed?. I think checkers.js is not descriptive enough. Also, appending the folder name to the filename will be nice. Check here for more details. - https://github.com/andela/engineering-playbook/tree/master/5.%20Developing/Conventions#commit-message
I will suggest you use `permission.middleware.js ` or any other name you can think of.